### PR TITLE
Rename internal/dinosaur to internal/central

### PR DESCRIPTION
**Pull request by dhaus67**
## Description

Rename the package from `internal/dinosaur` to `internal/central` because frankly, I don't understand why its still named this way :-)

Renaming was performed by using the Goland `Refactor` functionality, renaming the package of `internal/dinosaur` to `internal/central`.

Afterwards, verified that no import aliases were used by.

## Test manual

- E2E tests passing.

### CODE REVIEW

**Comment by dhaus67** [comment]
Current dependencies on/for this PR:
* main
  * **PR #433** <a href="https://app.graphite.dev/github/pr/stackrox/acs-fleet-manager/433" target="_blank">Graphite</a>  👈
    * **PR #434** <a href="https://app.graphite.dev/github/pr/stackrox/acs-fleet-manager/434" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/acs-fleet-manager/433?utm_source=stack-comment).

**Comment by misberner** [comment]
Could you briefly write in the PR description how you performed the renaming?

**Comment by dhaus67** [comment]
@misberner updated the description with the rename strategy.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ repos:
...
+      files: '(openapi/.*|pkg/workers/worker_interface.go|pkg/client/ocm/id.go|pkg/client/aws/client.go|pkg/client/ocm/client.go|pkg/client/iam/client.go|pkg/services/authorization/authorization.go|pkg/services/sso/iam_service.go|pkg/client/redhatsso/client.go|pkg/auth/auth_agent_service.go|internal/central/pkg/services/observatorium_service.go|internal/central/pkg/services/cluster_placement_strategy.go|internal/central/pkg/services/cloud_providers.go|internal/central/pkg/services/clusters.go|internal/central/pkg/services/quota.go|internal/central/pkg/services/fleetshard_operator_addon.go|internal/central/pkg/services/quota_service_factory.go|internal/central/pkg/clusters/cluster_builder.go|internal/central/pkg/clusters/provider.go|internal/central/pkg/services/dinosaur.go)'
```
Comment (misberner) [inline .pre-commit-config.yaml:11]: Does the goland refactor also recognize usages like this? Wow!

**Review by rukletsov — APPROVED** [review]
If I were you, I would have said `s/dinosaur//g` because frankly, stuff under `internal/dinosaur/pkg` is not necessarily related to Centrals, for instance, it includes API, configs, etc. But frankly this does not influence the thrust, so:

:shipit: Cretaceous–Paleogene time 🦕 🦖

**Review by misberner — APPROVED** [review]
Forgot to click approve in my earlier review

**Pull request closed without merging**